corehq/apps/callcenter/tasks.py

```diff
@@ -46,6 +47,18 @@ def calculate_indicators():
         indicator_set.get_data()
 
 
+def sync_web_user_usercases_if_applicable(user, spawn_task):
+    if settings.UNIT_TESTING:
+        return
+    for domain in user.get_domains():
+        domain_obj = Domain.get_by_name(domain)
+        if domain_obj.usercase_enabled and domain in USH_WEB_USER_CASE_CREATION.get_enabled_domains():
```

Comment:
`get_enabled_domains` isn't cached to it should be moved outside of the `for` loop. Alternately you could use `USH_WEB_USER_CASE_CREATION.enabled(domain)`